packages/runtime/test/transport/relationships.test.ts

```diff
@@ -996,6 +996,8 @@ describe("RelationshipDecomposition", () => {
         );
     });
 
+    let nameCounter = 0;
```

Comment:
Instead of the `nameCounter` I would prefer an approach in which two random character strings are generated for use as GivenName below.

Reply:
I suggested something like this instead of random characters such that if something fails, you have an idea which one failed